charts/node/README.md

```diff
@@ -22,6 +22,31 @@ helm install kusama-node parity/node --set node.chainDataSnapshotUrl=https://ksm
 ```
 ⚠️ For some chains where the local directory name is different from the chain ID, `node.chainPath` needs to be set to a custom value.
 
+### Optional Vault Integration
+
+To integrate this chart with vault:
+- Vault agent injector [installed](https://www.vaultproject.io/docs/platform/k8s/injector/installation) on the cluster
+- Kubernetes [auth enabled](https://learn.hashicorp.com/tutorials/vault/kubernetes-sidecar#configure-kubernetes-authentication) in your vault instance
+- Secrets for either the keys or the nodeKey created in your vault using kv-2
+- A policy for each of your secrets configured in your vault
+- an authentication role crated in your vault instance (should match the serviceAccount name for this chart) with policies for accessing your keys attached
+
+```
+node:
+  vault:
+    keys:
+      - name: aura
+        type: aura
+        scheme: sr25519
+        vaultPath: kv/secret/polkadot-node # path at which the secret is located in Vault
+        vaultKey: aura # key under which the secret value is stored in Vault
+        extraDerivation: "//${HOSTNAME}//aura" # allows to have unique derived keys for each pod of the statefulset
```

Comment:
Very useful.  Definitely an improvement being able to do this vs how we use `node.keys`